### PR TITLE
Update 1.8.0

### PR DESCRIPTION
**Pull request by kacperkrzyzak**

### CODE REVIEW

File: bundle/manifests/ibm-licensing-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -381,15 +381,15 @@ spec:
...
                     env:
                       - name: IBM_LICENSING_IMAGE
-                        value: quay.io/opencloudio/ibm-licensing:1.7.0
+                        value: quay.io/opencloudio/ibm-licensing:1.8.0
                       - name: IBM_LICENSE_SERVICE_REPORTER_UI_IMAGE
-                        value: quay.io/opencloudio/ibm-license-service-reporter-ui:1.7.0
+                        value: quay.io/opencloudio/ibm-license-service-reporter-ui:1.8.0
                       - name: IBM_POSTGRESQL_IMAGE
-                        value: quay.io/opencloudio/ibm-postgresql:12.0.8
+                        value: quay.io/opencloudio/ibm-postgresql:12.0.9
```
Comment (SzymonKowalczyk) [inline bundle/manifests/ibm-licensing-operator.clusterserviceversion.yaml:388]: Please change postrges to 1.8.0

File: bundle/manifests/ibm-licensing-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -623,16 +623,16 @@ spec:
...
+      image: quay.io/opencloudio/ibm-licensing-operator:1.8.0
     - name: IBM_LICENSING_IMAGE
-      image: quay.io/opencloudio/ibm-licensing:1.7.0
+      image: quay.io/opencloudio/ibm-licensing:1.8.0
     - name: IBM_LICENSE_SERVICE_REPORTER_UI_IMAGE
-      image: quay.io/opencloudio/ibm-license-service-reporter-ui:1.7.0
+      image: quay.io/opencloudio/ibm-license-service-reporter-ui:1.8.0
     - name: IBM_POSTGRESQL_IMAGE
-      image: quay.io/opencloudio/ibm-postgresql:12.0.8
+      image: quay.io/opencloudio/ibm-postgresql:12.0.9
```
Comment (SzymonKowalczyk) [inline bundle/manifests/ibm-licensing-operator.clusterserviceversion.yaml:632]: Please change postrges to 1.8.0

File: config/manager/manager.yaml [hunk 3]
```diff
@@ -35,15 +35,15 @@ spec:
...
           env:
             - name: IBM_LICENSING_IMAGE
-              value: quay.io/opencloudio/ibm-licensing:1.7.0
+              value: quay.io/opencloudio/ibm-licensing:1.8.0
             - name: IBM_LICENSE_SERVICE_REPORTER_UI_IMAGE
-              value: quay.io/opencloudio/ibm-license-service-reporter-ui:1.7.0
+              value: quay.io/opencloudio/ibm-license-service-reporter-ui:1.8.0
             - name: IBM_POSTGRESQL_IMAGE
-              value: quay.io/opencloudio/ibm-postgresql:12.0.8
+              value: quay.io/opencloudio/ibm-postgresql:12.0.9
```
Comment (SzymonKowalczyk) [inline config/manager/manager.yaml:42]: Please change postrges to 1.8.0

**Review by SzymonKowalczyk — CHANGES_REQUESTED** [review]

File: bundle/manifests/ibm-licensing-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (kacperkrzyzak) [reply bundle/manifests/ibm-licensing-operator.clusterserviceversion.yaml:388]: The current version for Postgres is 12.0.9
Do we want to change versioning strategy for our Postgres image?

Comment (SzymonKowalczyk) [reply bundle/manifests/ibm-licensing-operator.clusterserviceversion.yaml:388]: We agree with @pgodowski to have all the operand images in the same version

**Review by SzymonKowalczyk — APPROVED** [review]

**Pull request merged**